### DISCUSSION TITLE
We support AzerothCore financially

### DISCUSSION THREAD

**Discussion by ChromieCraftServer** [azerothcore/azerothcore-wotlk#5576] [General]
At [ChromieCraft.com](https://www.chromiecraft.com) we support AzerothCore financially. We are investing the donations coming from our player base into AzerothCore open-source development.

Whether you like our game-style or not, support us and we will make AzerothCore stronger. The more successful ChromieCraft will be, the stronger AzerothCore will get.

You can support us in many ways, even just by inviting your friends to play on our realm, sharing our website around, etc...

https://www.chromiecraft.com/how-to-contribute

ChromieCraft is a non-profit project created by the founders of AzerothCore, for the AzerothCore.

**Comment by ChromieCraftServer** [comment]
https://www.chromiecraft.com/sponsoring-azerothcore/

**Comment by htc16** [comment]
This server is really good, I recommend it!
